iOS/MMIWG2S/Views/PageIndicator.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  PageIndicator.swift
+//  MMIWG2S
+//
+//  Created by Callum Johnston on 8/11/20.
+//  Copyright © 2020 Google LLC. All rights reserved.
+//
+
+import UIKit
+
+class PageIndicator: UIButton {
```

Comment:
It'd be cool to use this :)

Reply:
I am using the PageViewController.  So far its ok, but maybe we can talk about substituting with this in the future.